### PR TITLE
feat: add stopPropagation api to timepicker

### PR DESCRIPTION
**Pull request by YannLynn**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [x] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [ ] Bugfix
 - [X] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### Changelog
🇨🇳 Chinese
- Feat: Timepicker 新增 stopPropagation 用于判断是否阻止弹出层上的点击事件冒泡 

---

🇺🇸 English
- Feat: Timepicker adds stopPropagation to determine whether to prevent click events on the popup layer from bubbling up.

### Checklist
- [ ] Test or no need
- [ ] Document or no need
- [ ] Changelog or no need

### Other
- [ ] Skip Changelog

### CODE REVIEW

File: packages/semi-ui/timePicker/TimePicker.tsx [hunk 1]
```diff
@@ -470,8 +473,10 @@ export default class TimePicker extends BaseComponent<TimePickerProps, TimePicke
             triggerRender,
             motion,
             autoAdjustOverflow,
+            stopPropagation,
             ...rest
         } = this.props;
+        console.log('stopPropagation', stopPropagation);
```
Comment (pointhalo) [inline packages/semi-ui/timePicker/TimePicker.tsx:479]: 这log咋还留着呢

File: packages/semi-ui/timePicker/TimePicker.tsx [hunk 2]
```diff
@@ -82,6 +82,7 @@ export type TimePickerProps = {
     secondStep?: number;
     showClear?: boolean;
     size?: InputSize;
+    stopPropagation: boolean | string;
```
Comment (pointhalo) [inline packages/semi-ui/timePicker/TimePicker.tsx:85]: string 是啥玩法，不是应该只有 true or false吗

**Comment by xmsz-stu** [comment]
stopPropagation的类型现在是必选了

**Comment by YannLynn** [comment]
> stopPropagation的类型现在是必选了

版本 2.50.0-beta.0 更正了这个问题哈。

**Pull request merged**